lib/kubeclient/common.rb

```diff
@@ -227,14 +227,11 @@ def create_entity(entity_type, entity_config, klass)
     end
 
     def update_entity(entity_type, entity_config)
-      name = entity_config.metadata.name
-      # to_hash should be called because of issue #9 in recursive open
-      # struct
-      hash = entity_config.to_hash
-      ns_prefix = build_namespace_prefix(entity_config.metadata['table'][:namespace])
+      name      = entity_config[:metadata][:name]
```

Comment:
> but seemed like the preference was towards using [] rather than method references

yes, because we are aware with override specific issues of the 'namespace' part :)
but it's ok.